src/trace_processor/util/args_utils.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef SRC_TRACE_PROCESSOR_UTIL_ARGS_UTILS_H_
+#define SRC_TRACE_PROCESSOR_UTIL_ARGS_UTILS_H_
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "perfetto/base/status.h"
+#include "src/trace_processor/types/variadic.h"
+
+namespace perfetto::trace_processor {
+class TraceStorage;
+
+class ArgSet;
+
```

Comment:
AddJsonValueToArgs should also be moved to this file since you're doing this.

Reply:
I looked into it and I think it's better off in json_utils.h as it's very specific to parsing JSON (e.g. requires jsoncpp, etc).